tasmota/xdrv_27_shutter.ino

```diff
@@ -111,21 +116,23 @@ int32_t ShutterPercentToRealPosition(uint32_t percent, uint32_t index)
       if (0 == Settings.shuttercoeff[j][index]) {
         AddLog_P2(LOG_LEVEL_ERROR, PSTR("SHT: RESET/INIT CALIBRATION MATRIX DIV 0"));
         for (uint32_t k = 0; k < 5; k++) {
-          Settings.shuttercoeff[k][index] = SHT_DIV_ROUND(calibrate_pos[k+1] * 1000, calibrate_pos[5]);
+          Settings.shuttercoeff[k][index] = (uint32_t)calibrate_pos[k+1] * 1000 / calibrate_pos[5];
```

Comment:
When not using SHT_DIV_ROUND here we will introduce an old bug again that shutter might not reach the position ask in command ShutterPosition.